Add ?warn command (#6)
codewars/discord-bot: text/warn/inappropriate.md

You've violated the following rule(s):

- Treat everyone with respect. Absolutely no harassment, witch hunting, sexism, racism, or hate speech will be tolerated.

> - No NSFW or obscene content. This includes text, images, or links featuring nudity, sex, hard violence, or other graphically disturbing content.

Reviewer: Should we split this up into two messages? The two points are kinda unrelated and just makes for. I propose `conduct` and `content` for the identifiers.

Reviewer 2: Yeah, I agree with splitting.
For identifiers, I want something descriptive for others to understand what the warning was about. These messages are DMed to the target user.

Reviewer 2: Actually, `conduct` and `content` might be nice and short.

> You are receiving this message because you have been found posting inappropriate content on the Codewars Discord server.
> 
> Please take a moment to be reminded of the following rules from this server's #readme channel:
> 
> - Treat everyone with respect. Absolutely no harassment, witch hunting, sexism, racism, or hate speech will be tolerated.
> - No NSFW or obscene content. This includes text, images, or links featuring nudity, sex, hard violence, or other graphically disturbing content.
> 
> Consider this a formal warning. If you insist on engaging in such behavior, we reserve the right to kick or ban you from the server indefinitely, and your Codewars account (if any) may be restricted.

Reviewer 2: ```suggestion
You've violated the following rule(s):

- Treat everyone with respect. Absolutely no harassment, witch hunting, sexism, racism, or hate speech will be tolerated.
- No NSFW or obscene content. This includes text, images, or links featuring nudity, sex, hard violence, or other graphically disturbing content.

If you insist on engaging in such behavior, we reserve the right to kick or ban you from the server indefinitely, and your Codewars account (if any) may be restricted. You've been warned.
```
